interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+<?php   
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="claims";
+    require_once "../../../../globals.php";
+   
+    require_once '../src/ClaimSearchModel.php';
+    require_once '../src/ClaimSearch.php';
+    require_once '../src/ClaimRevApi.php';
+    require_once '../src/UploadEdiFileContentModel.php';
```

Comment:
should remove all 4 lines above, since the autoloader in globals.php will bring in all your classes

Reply:
done